DoppelVision/DoppelVision.json

```diff
@@ -0,0 +1,311 @@
+{
+    "type": "FolderSyncDefinition",
+    "name": "DoppelVision",
+    "description": "Comprehensive overview of digital risk protection metrics and alerts",
+    "children": [
+        {
+            "type": "SavedSearchWithScheduleSyncDefinition",
+            "name": "Alerts By Product",
+            "search": {
+                "queryText": "$$logsrc \n| json field=_raw \"product\"\n| timeslice 1d\n| count by _timeslice, product\n| sort by _timeslice asc\n| transpose row _timeslice column product as * ",
```

Comment:
I would recommend parsing the JSON at ingest time using FER so that you don't have to parse json at query time
this will drastically increase performance and also allow customer to custom use those field for their searches if they want to without using explicit parse json
Also this would speed up your queries for dashboard

Reply:
does this change requires any change in field extraction rules?
if yes, shouldn't they be updated in readme file?

Reply:
Sauarbh, This does not require any change in field extraction rule, we were successfully able to get the data in dashboard with the existing extraction rule and with parsing removed at query time.